modules/services/eks/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "region" {
+  description = "The AWS region where clusters are located"
+  type        = string
+}
```

Comment:
We'll need the region for `provider "awscc" {}` block unless it will be dynamically injected into the main TF code, right?

<img width="761" alt="Screenshot 2024-02-08 at 13 33 12" src="https://github.com/draios/terraform-aws-secure-for-cloud/assets/142799039/1aad0e70-aa03-49e3-acfe-8903dbcc52dd">

Reply:
> I believe this can be determined dynamically at run time

Yes, you are right - it's "implicit provider inheritance":
https://developer.hashicorp.com/terraform/language/modules/develop/providers#implicit-provider-inheritance

The only one we'll need to keep will be `awscc`.

Reply:
As you pointed out via chat, the resource I linked does not currently support `access_policies`.
Scratch this then, but `awscc` provider must move to the calling script anyway.

Reply:
Sounds good - I'll remove the providers file then.